### ISSUE TITLE
Refund

### ISSUE THREAD

**Issue by damonbaneharrison** [FrozenIronSoftware/Twitched#61]
I would like a refund now that Twitched has been removed from my Roku device.

**Comment by roadriverrail** [comment]
Or at least an explanation

**Comment by Cruv** [comment]
Yeah its honestly ridiculous how this all went down. Without any word from Roku or the Twitched dev there is literally no alternative other than moving to another device (which doesn't fit my use case as private listening is a big thing in my house). Really unprofessional.

**Comment by ryancole** [comment]
I'm trying to contact Roku about a refund.
